rdf_differ/adapters/diff_adapter.py

```diff
@@ -244,58 +172,77 @@ def create_dataset(self, dataset_name: str) -> tuple:
             'dbName': dataset_name
         }
 
-        response = requests.post(urljoin(self.triplestore_service_url, f"/$/datasets"),
-                                 auth=requests.auth.HTTPBasicAuth('admin', 'admin'),
-                                 data=data)
+        response = self.http_requests.post(urljoin(self.triplestore_service_url, f"/$/datasets"),
+                                           auth=HTTPBasicAuth('admin', 'admin'),
```

Comment:
Don't think that it's a good idea to have the auth here, maybe in the future this `http_requests` actually be an instance that handles this username and password.
Something like 
```
class HTTPClient:
   def __init__(self, username=None, password=None):
      ...
   def post(endpoint, data):
      if self.username and self.password:
         return requests.post(...., auth=(self.username, self.password)
```

Reply:
At least the user and password can be taken from the config.py